examples/verify-quorum/host/src/main.rs

```diff
@@ -155,3 +154,12 @@ async fn main() -> eyre::Result<()> {
     println!("successfully generated and verified proof for the program!");
     Ok(())
 }
+
+// We can't use `public_key_to_address()` from `reth_primitives`` because Reth depend on
+// `secp256k1` 0.30 while we are still on 0.29.
```

Comment:
```suggestion
// Can't use `public_key_to_address()` from `reth_primitives` because Reth depends on
// `secp256k1` 0.30 while this crate is still on 0.29.
```